.github/workflows/deploy-digitalocean.yml

```
@@ -0,0 +1,322 @@
+on:
+  workflow_run:
+    workflows: ["Docker Build"]
+    types:
+      - completed
+    branches:
+      - main
+      - develop
+
+  workflow_dispatch:
+    inputs:
+      environment:
```

actionlint (1.7.10)

[error] input type of "environment" is "choice" but "options" is not set

(events)